js/redux/store.js

```diff
@@ -1,14 +1,22 @@
-import { createNavigationEnabledStore } from '@expo/ex-navigation';
-import { compose, createStore, applyMiddleware } from 'redux';
-import thunk from 'redux-thunk';
-import rootReducer from './reducers';
+import { createNavigationEnabledStore } from "@expo/ex-navigation";
```

Comment:
If you are using react-navigation, why are you importing this component from ex-navigation? You should not.